plugins/devo/help.md

```diff
@@ -168,7 +168,7 @@ _This plugin does not contain any custom output types._
 
 # Version History
 
-* 2.0.0 - Configuration for interval input parameter
+* 2.0.0 - Configuration for interval input parameter, changed way of sending alerts by trigger 
```

Comment:
Which version should i use? Stay with this or add `3.0.0`?

Reply:
Hey @lukaszlaszuk Because the output schema has changed, it will not be backward-compatible in the workflow builder. A major version bump to 3.0.0 is correct. You'll want to add a new line for 3.0.0 in the version history.

On a separate note, can we improve this error message to be more customer-focused. E.g. It doesn't say anything about what it solves which is fixing an issue in the Get New Alerts trigger that was creating duplicated alerts (..in some config/context) and also call out that the output changed.

Thanks!

Reply:
Sure, i added version history message for that change